python/etl/commands.py

```diff
@@ -1127,9 +1127,11 @@ def callback(self, args):
         max_concurrency = args.max_concurrency or etl.config.get_config_int(
             "resources.RedshiftCluster.max_concurrency", 1
         )
-        statement_timeout = args.statement_timeout or etl.config.get_config_int(
-            "resources.RedshiftCluster.statement_timeout", 0
-        )
+        statement_timeout = args.statement_timeout
+        if etl.config.is_set("resources.RedshiftCluster.statement_timeout"):
+            statement_timeout = statement_timeout or etl.config.get_config_int(
+                "resources.RedshiftCluster.statement_timeout"
+            )
```

Comment:
When typing this:
```
arthur.py load --statement-timeout 0
```
I would expect the timeout to be set to 0 (effectively, disabled) since I bothered to set something on the commandline. The code however falls back to the setting.

```suggestion
        if statement_timeout is None and etl.config.is_set("resources.RedshiftCluster.statement_timeout"):
            statement_timeout = etl.config.get_config_int(
                "resources.RedshiftCluster.statement_timeout"
            )
```

Also, `get_config_int` errors out with a default of None. This simplifies type checking a lot. If we had an additional `get_config_optional_int` though, the code would simplify further.